pinot-broker/src/main/java/com/linkedin/pinot/broker/requesthandler/BrokerRequestHandler.java

```diff
@@ -543,6 +547,7 @@ private BrokerResponse processOptimizedBrokerRequests(@Nonnull BrokerRequest ori
       _brokerMetrics.addMeteredTableValue(rawTableName, BrokerMeter.BROKER_RESPONSES_WITH_PARTIAL_SERVERS_RESPONDED,
           1L);
     }
+    _brokerMetrics.addMeteredQueryValue(originalBrokerRequest, BrokerMeter.RESPONSE_SIZE_UPPER_BOUND, totalResponseSize.longValue());
```

Comment:
Can we rename the metric to be totalServerResponseSize or something more explicit like this? UPPER_BOUND is not so intuitive to understand.

Reply:
Good idea 👍